module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,13 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> list) {
+    list.sort();
+    std::deque<std::string> deq{list.begin(), list.end()};
+    std::sort(deq.begin(), deq.end(), [](const std::string& first, const std::string& second) {
+        return first.length() < second.length();
+    });
```

Comment:
Our tests suck because they don't detect that your implementation is not entirely correct :)
What about the lexicographical part when the length is the same?

Reply:
crrected